proto/mls/message_contents/content.proto

```diff
@@ -52,7 +52,35 @@ message PlaintextEnvelope {
     string idempotency_key = 2;
   }
 
+  // Version 2 of the encrypted envelope
+  message V2 {
+    oneof message_type {
+      // Expected to be EncodedContent
+      bytes content = 1;
+      // Initiator sends the request to receive history
+      MessageHistoryRequest message_history_request = 2;
+      // Other credentialed party sends response
+      MessageHistoryResponse message_history_response = 3;
+    }
+  }
```

Comment:
Should we be adding message backups for V1 messages as well?

Reply:
Perfectly good question!  that I don't know the answer to.  
This is very much a first pass to get a basic round-trip request<>response functioning. 
If we need to backport and/or add additional support for `V1` as well, we can absolutely do that.